packages/p2p/src/stores/sendbird-store.ts

```diff
@@ -304,7 +304,6 @@ export default class SendbirdStore extends BaseStore {
             if (!this.messages_ref?.current) return;
 
             if (this.messages_ref.current.scrollTop === 0) {
-                this.setIsChatLoading(true);
```

Comment:
why remove this? shouldn't we keep this so we can show the loading screen

Reply:
No @ameerul-deriv, @nada-deriv pointed out to me that when ever a user scrolls, the loader shows up, they're not necessary

Reply:
ahhh thanks for clarifying 🙏